modules/k6/src/main/java/org/testcontainers/k6/K6Container.java

```diff
@@ -0,0 +1,98 @@
+package org.testcontainers.k6;
+
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.MountableFile;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class K6Container extends GenericContainer<K6Container> {
+
+    /** Standard image for k6, as provided by Grafana. */
+    public static final DockerImageName K6_IMAGE = DockerImageName.parse("grafana/k6:0.49.0");
```

Comment:
version should be defined by the user. This is just to make sure K6Container is tested and compatible with `grafana/k6`

```suggestion
    private static final DockerImageName K6_IMAGE = DockerImageName.parse("grafana/k6");
```